.github/actions/deploy-to-control-plane/action.yml

```
@@ -1,56 +1,125 @@
-          ${{ runner.os }}-docker-
+        script: |
+          // Create GitHub deployment
+          const deployment = await github.rest.repos.createDeployment({
+            owner: context.repo.owner,
+            repo: context.repo.repo,
+            ref: context.sha,
+            environment: 'review-app',
+            auto_merge: false,
+            required_contexts: []
+          });
+          
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)